### ISSUE TITLE
Login asks for too much rights

### ISSUE THREAD

**Issue by dbuenzli** [avsm/mirage-ci#8]
I'm not going to grant you write access to my repos to login:

```
Repositories
Public and private
This application will be able to read and write all public and private repository data. This includes the following:

Code
Issues
Pull requests
Wikis
Settings
Webhooks and services
Deploy keys
```

**Comment by talex5** [comment]
@avsm: you can adjust the scopes it asks for now with this:

https://github.com/moby/datakit/blob/97b3d230535397a813323902c23751e176481a86/ci/src/datakit_ci.mli#L353

**Comment by talex5** [comment]
Fixed in #33.
